source/modules/collection-api/account/overview.md.erb

```diff
@@ -1,19 +1,18 @@
 ## Account Activity
 
-The following functions will be used to relay actions associated with an account:
+You can send account-related information to Inspetor via the REST API. The `/accounts` endpoint supports `POST` (create), `PUT` (update), and `DELETE` (delete) operations with the following arguments:
 
-- <a href="#account-creation">trackAccountCreation</a>
-- <a href="#account-updates">trackAccountUpdate</a>
-- <a href="#account-deletion">trackAccountDeletion</a>
+Property               | When is it required?   | Type                | Description
+---------------------- | ---------------------  | ----                | -----------
+**id**                 | `POST`, `PUT`, `DELETE`| String              | Your unique identifier for this account in your platform
+**email**              | `POST`                 | String              | The email associated with the account
+**document**           |                        | String              | The CPF number associated with the account. The expected format is only numbers, no dots and dashes.
```

Comment:
Nit: Why some stuff is in bold and others not? I thought we were doing a distinction between required fields and not required fields, but this does not look like the case

Reply:
yeah, I think we should standardize